### PR TITLE
Added GPU course

### PR DESCRIPTION
**Pull request by mondus**
Added GPU course event

### CODE REVIEW

File: _events/gpucourse-2022-02-07.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+speaker: "Paul Richmond"
+---
+
+## 12 Week Parallel Computing with GPUs course
+
+Starting this term (Feb) is a 12 week course on "Parallel Computing with GPUs". The course has been designed as an undergraduate 4th year (and MSc) module for Computer Science but is available to staff and PhD Students (as part of the DDP program) and regularly has staff and PhD student enrolment.
+
+The first 3 weeks are focused on teaching C and OpenMP followed by GPU programming with CUDA C. Lectures are provided as pre-recorded mini lectures and there are 2 hours of scheduled support per week to undertake practical lab classes. Assessment is not required for DDP or staff participants. You can attend all or just parts of the course.
+
+If you are interested in enrolling as staff or a PhD student then please express you interest on the following google form: bit.ly/GPU2022
```
Comment (Robadob) [inline _events/gpucourse-2022-02-07.md]: Will jekyll manage to autolinkify this?

Comment (mondus) [reply _events/gpucourse-2022-02-07.md]: Probs not

Comment (mondus) [inline _events/gpucourse-2022-02-07.md]:
```suggestion
If you are interested in enrolling as staff or a PhD student then please express you interest on the following google form: [bit.ly/GPU2022](bit.ly/GPU2022)
```

Comment (mondus) [reply _events/gpucourse-2022-02-07.md]: Have fixed

**Review by Robadob — APPROVED** [review]

**Review by bobturneruk — APPROVED** [review]
Looking good!

**Pull request merged**